_posts/zheinsohn/12-2-21-meetup-writeup-zheinsohn

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+author: zheinsohn
+title: Zoe's Meetup Writeup
```

Comment:
Could you enclose this title in quotes? The apostrophe will cause problems, otherwise.